src/dataunion/DataUnion.ts

```diff
@@ -296,7 +296,7 @@ export class DataUnion {
         return +adminFeeBN.toString() / 1e18
     }
 
-    async getAdminAddress(): Promise<Todo> {
+    async getAdminAddress(): Promise<string> {
```

Comment:
did we have the EthereumAddress type alias now?

Reply:
Fixed